components/slidingPanel/slidingPanelHeader.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const SlidingPanelHeader = ({ title }) => {
+  if (!title) {
+    return <noscript />;
```

Comment:
It was `null` in my first implementation. But then I stuck with testing, it causes [this](https://github.com/facebook/jest/issues/642) error. Seems like unless we migrated to React 15 we still need to use `noscript`. By the way, I've searched other components and found many places where we also use `noscript`. Once we migrated I think could be nice to get rid of them.